### DISCUSSION TITLE
is this one a bug or not ?

### DISCUSSION THREAD

**Discussion by billychen2014** [appium/appium#15134] [Q&A]
![image](https://user-images.githubusercontent.com/7981655/108973511-6299a680-76bf-11eb-8ce2-a33279acbec7.png)

hello,

please check the red zone part ,  which value of 'accesssibility id'  was shown as  "自动化,,"  which has extra ",," 
testing device is iPhone xs max (13.6 OS),

is this one potential bug or something i did wrong ?

**Comment by KazuCocoa** [comment]
I think it is your app implemented so for iOS.
Please check the result of [page source](http://appium.io/docs/en/commands/session/source/) command. If the result also has the same issue, it will be your implement stuff.
